app/controllers/v0/sessions_controller.rb

```diff
@@ -131,6 +134,10 @@ def stats(status, saml_response = nil, failure_tag = nil)
         StatsD.increment(STATSD_LOGIN_NEW_USER_KEY) if request_type == 'signup'
         StatsD.increment(STATSD_SSO_CALLBACK_KEY,
                          tags: ['status:success', "context:#{saml_response.authn_context}"])
+        # track users who have a shared sso cookie
+        StatsD.increment(STATSD_SSO_SHARED_COOKIE,
```

Comment:
We seem to. be initializing all the StatsD counters in `/config/initializers/statsd.rb`. These also appear to spit out a bunch of warnings on app startup but I think the alternative is that we get some weird behavior in prometheus with uninitialized counters.

Reply:
@patrickvinograd did we need something else other than this?  https://github.com/department-of-veterans-affairs/vets-api/pull/3762/files#diff-da38dd1d4d165deac7ffeeb8f413bbdeR16